lib/common/nvpair.c

```diff
@@ -194,7 +194,7 @@ pcmk__nvpair_add_xml_attr(gpointer data, gpointer user_data)
 /*!
  * \brief Add XML attributes based on a list of name/value pairs
  *
- * \param[in]     list  List of name/value pairs
+ * \param[in,out] list  List of name/value pairs
```

Comment:
This was one of those where I'm unsure about the library. glib declares g_slist_foreach() without const, even though presumably it should be. I've gone back and forth in such cases between "it's obviously [in]" and "use [in,out] because they might change the implementation".

The root problem is that const is annoying to think about every time you write a function, so few people bother, and other libraries' laziness makes it difficult for us to stop being lazy :)